{"name": "test_create_order_with_invalid_ingredient", "status": "failed", "statusDetails": {"message": "AssertionError: assert 403 == 500\n +  where 403 = <Response [403]>.status_code", "trace": "self = <test_create_order.TestCreateOrder object at 0x102989cd0>\n\n    @allure.feature(\"Создание заказа\")\n    @allure.story(\"Попытка создать заказ с невалидными ингредиентами\")\n    def test_create_order_with_invalid_ingredient(self):\n        order_data = {\"ingredients\": [\"invalid_hash\"]}\n        headers = {\"Authorization\": \"Bearer <accessToken>\"}\n        response = requests.post(f\"{BASE_URL}/orders\", json=order_data, headers=headers)\n>       assert response.status_code == 500  # Ошибка: невалидный хеш ингредиента\nE       assert 403 == 500\nE        +  where 403 = <Response [403]>.status_code\n\ntest_create_order.py:46: AssertionError"}, "start": 1732755573750, "stop": 1732755574038, "uuid": "6030e72b-7cb7-4ce2-980e-2573c659cef3", "historyId": "9bf7d65d22acbc46bdc4ddb4293ba878", "testCaseId": "9bf7d65d22acbc46bdc4ddb4293ba878", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_with_invalid_ingredient", "labels": [{"name": "feature", "value": "Создание заказа"}, {"name": "story", "value": "Попытка создать заказ с невалидными ингредиентами"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "admins-MacBook-Pro.local"}, {"name": "thread", "value": "24154-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}